backend/api/Services/RobotService.cs

```diff
@@ -28,5 +40,32 @@ public async Task<IEnumerable<Robot>> ReadAll()
         {
             return await _context.Robots.FirstOrDefaultAsync(robot => robot.Id.Equals(id, StringComparison.Ordinal));
         }
+
+        public async Task<Report> StartMission(Robot robot, string missionId)
+        {
+            var response = await _isarService.StartMission(missionId);
+            if (!response.IsSuccessStatusCode)
+            {
+                throw new MissionException($"Could not start mission with id: {missionId}");
+            }
```

Comment:
Should you log some content from the response if this is the case? Like the status code, reason and/or any relevant response from ISAR?

Reply:
Logging to be taken care of by:
https://github.com/equinor/flotilla/issues/199